src/js/components/Select/__tests__/SelectUncontrolled-test.js

```diff
@@ -28,39 +41,40 @@ describe('Select', () => {
     expect(component.toJSON()).toMatchSnapshot();
   });
 
-  test('0 value', () => {
-    const component = renderer.create(
-      <Select
-        id="test-select"
-        placeholder="test select"
-        options={[0, 1]}
-        value={0}
-      />,
-    );
-    expect(component.toJSON()).toMatchSnapshot();
-  });
-
-  test('opens', done => {
-    window.scrollTo = jest.fn();
+  test('opens and closes', () => {
+    jest.useFakeTimers();
+    const onOpen = jest.fn();
+    const onClose = jest.fn();
     const { getByPlaceholderText, container } = render(
       <Select
         placeholder="test select"
         id="test-select"
         options={['one', 'two']}
+        onOpen={onOpen}
+        onClose={onClose}
       />,
     );
     expect(container.firstChild).toMatchSnapshot();
     expect(document.getElementById('test-select__drop')).toBeNull();
 
     fireEvent.click(getByPlaceholderText('test select'));
-
     expect(container.firstChild).toMatchSnapshot();
     expectPortal('test-select__drop').toMatchSnapshot();
+    // advance timers so the select opens
+    jest.advanceTimersByTime(100);
+    // verify that select is open
+    expect(document.activeElement).toMatchSnapshot();
+    expect(onOpen).toHaveBeenCalled();
```

Comment:
Let's avoid `toHaveBeenCalled` and use `toHaveBeenCalledTimes` instead for more accuracy. true for the rest of the PR.

Reply:
Changed in the most recent push